### DISCUSSION TITLE
Receiving an error- malloc(): invalid size (unsorted)

### DISCUSSION THREAD

**Discussion by jfpe37** [firemodels/fds#13383] [General]
Good morning,
I am a new user of a Linux computer
During several different runs I receive errors that I do not recognize when running on Windows computers.
When running a simulation of burning an electric car I got the following error: **_malloc(): invalid size (unsorted)_** after 239 seconds blowers start working after 100 seconds.
In terms of memory, the computer has 1 terabyte, and i am using FDS 6.9.1.
I would appreciate your help,
Why do I receive errors of this type? Is it related to the Linux settings? For the way of installing the software maybe?
On a Windows computer I didn't have this type of errors and it has been running without problems.

I am attaching the error that was received and the FDS after I removed building details from it due to maintaining confidentiality.

[HHS-3A_002_TEST.txt](https://github.com/user-attachments/files/16844367/HHS-3A_002_TEST.txt)
![Screenshot from 2024-09-03 08-51-03](https://github.com/user-attachments/assets/09a45d15-8b7d-451e-a235-a448d2c48fb0)

Thanks in advance

**Comment by drjfloyd** [comment]
I don't see anything obviously in error with the input lines you attached. Without the complete input file, we can't run the case to try and get the same error.  Your file header shows you are using an old version of Pyrosim and therefore likely an old version of FDS. Try the latest version.

**Reply by jfpe37** [reply]
thanks
I updated the version of Pyrosim and ran again. 
The FDS version installed on my Linux computer is the latest version 6.9.1
I will wait for the results.
